src/components/farm/Farm.tsx

```diff
@@ -28,7 +28,7 @@ export default function Farm() {
           <>
             <FarmLastAttemptedProof />
             <FarmLatestBlockChallenges />
-            <AdvancedOptions>
+            <AdvancedOptions expanded>
```

Comment:
advanced options are collapsed by default

Reply:
let me fix that
my ide auto added that sorry.